spacy/errors.py

```diff
@@ -209,7 +209,10 @@ class Warnings(metaclass=ErrorsWithCodes):
             "`enabled` ({enabled}). Be aware that this might affect other components in your pipeline.")
     W124 = ("{host}:{port} is already in use, using the nearest available port {serve_port} as an alternative.")
 
+    # v4 warning strings
     W400 = ("`use_upper=False` is ignored, the upper layer is always enabled")
+    W401 = ("`incl_prior is True`, but the selected knowledge base type {kb_type} doesn't support prior probability "
+            "lookups.")
```

Comment:
We should add here what this will mean in practice: prior probabilities will in fact not taken into account, even if the KB has them available - if it does it should change the flag to `True`. Maybe we should even say that specifically, that if it's a custom KB that does support prior probabilities they need to change that method. If not, `incl_prior` simply won't work.

Reply:
Updated in https://github.com/explosion/spaCy/pull/12341/commits/2377b67f81d5707f46ba6ca085591dcc9256f334. Feel free to close if you think that's good.